### ISSUE TITLE
compiling issues

### ISSUE THREAD

**Issue by MOAMaster** [obiwanjacobi/vst.net#55]
**Describe the bug**
I can't get the whole application to compile in VS 2019. I get errors.
Severity	Code	Description	Project	File	Line	Suppression State
Error	C1083	Cannot open include file: 'pluginterfaces/base/conststringtable.h': No such file or directory	Jacobi.Vst3.Interop	D:\VSTPlugins2019\vst.net\Source3\Code\Jacobi.Vst3.Interop\usediids.cpp	11	

I get these errors for every include file in usediids.cpp.

These header files are from vst3. So, I copied a version of
vst3 into the folder where your code resides.
Still couldn't find them.

I then went to the interop project properties and replaced your
C++ include files with a hard coded path to the location of the
header files on the vst3 libraries.
Still can't seem to find the header files.
Then, I hardcoded the path to the header files in usedidds.cpp.
Then, I get errors that tell me those header files can't find other header
files that they reference internally.

How do I get your stuff to build?

I noticed that the interop project references a lib folder with the header files. But, you did not upload this lib folder.

Thanks.

Bruce Krell

**Comment by obiwanjacobi** [comment]
VST.NET for VST3 is not finished yet - some of it works. 
You have to download the VST3 SDK from the Steinberg web site. 
That should contain all the files you need. 
The Steinberg VST3 SDK license does not allow redistribution, so I cannot upload it here.

**Comment by MOAMaster** [comment]
I had the VST3 SDK source which I built.

Then, I copied the DLLs into a lib folder in the folder where I installed your stuff.

I also copied the header files into the lib folder since that is where your referenced them.

I changed the path to that folder on my hard drive.


I still get those error messages about not finding the header files.

 

There is some configuration problem with the stuff you placed on github.
